test/index.js

```diff
@@ -0,0 +1,38 @@
+var lastChar = require('../lib')
+  , assert = require('assert')
+  ;
+
+describe('last-char', function () {
```

Comment:
You should create your own shareable config for `eslint`. It'd make it easier to contribute and enforce devs to follow your style. Set the linting script as a pre-commit hook and voila. 

Have you heard about it?

You just set your eslint config and publish it as an npm module, and tell eslint to use it while linting.

http://eslint.org/docs/developer-guide/shareable-configs